lib-injection/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM scratch AS dotnet_build
+ARG LINUX_PACKAGE
+ADD ${LINUX_PACKAGE} /
```

Comment:
For any reviewers wondering, `ADD` differs from `COPY` in that it automatically extracts the   `tar.gz` file

Reply:
I added a small comment with this content in 641338a31ffb1e2ea7c60d6ab6653901efb5624e